### PR TITLE
add support to UPS negotiate rates

### PR DESCRIPTION
**Pull request by gtuxyco**
Support for the negotiate UPS negotiate rates.

### CODE REVIEW

**Comment by helmutschneider** [comment]
Thanks for the PR! Please write a better description. I've never personally used this feature of UPS so please explain what it does.

File: src/UPS/Service.php [hunk 1]
```diff
@@ -176,9 +176,7 @@ public function getQuotes(QuoteRequest $request): PromiseInterface
                             ],
                         ];
                     }, $parcels),
-                    //'ShipmentRatingOptions' => [
-                    //    'NegotiatedRatesIndicator' => '',
-                    //],
+                    empty($this->credentials->getShipperNumber()) ?: 'ShipmentRatingOptions' => ['NegotiatedRatesIndicator' => '',],
```
Comment (helmutschneider) [inline src/UPS/Service.php:179]: This is too complex for my taste. What happens to the array if the empty-invokation returns true? Do you get an empty null-value in the array?

Comment (gtuxyco) [reply src/UPS/Service.php:179]: You are right is too complex... let me edit it

**Comment by gtuxyco** [comment]
So you need to pass the account number and the flag to get the negotiate rates. i add another field to the ups credential, when you pass the account number i will add the flag
ups response and additional field.
                "TotalCharges": {
                    "CurrencyCode": "USD",
                    "MonetaryValue": "" -> original
                },
                "NegotiatedRateCharges": {
                    "TotalCharge": {
                        "CurrencyCode": "USD",
                        "MonetaryValue": "" -> negotiated 
                    }

**Comment by gtuxyco** [comment]
> Thanks for the PR! Please write a better description. I've never personally used this feature of UPS so please explain what it does.

Basically if you have businesses account with ups, they will give you discounted prices. but is not like DHL they need to have a flag in the request and they response the 2 prices. the original and you discounted price

File: composer.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
     ],
     "minimum-stability": "stable",
     "require": {
-        "php": ">=7.4.0",
+        "php": ">=7.3.0",
```
Comment (helmutschneider) [inline composer.json:14]: This is not OK as we are using arrow functions in this lib.

**Pull request closed without merging**